src/main/java/Duke.java

```diff
@@ -1,10 +1,27 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void startDuke(){
+        CommandManager greetingCommand = new CommandManager();
+        greetingCommand.sayHello();
+    }
+
+    public static void useDuke(){
+        UI readInputCommand = new UI();
+        CommandManager commandManager = new CommandManager();
+        //Only use one scanner object, otherwise there will be errors
```

Comment:
This comment sounds like it is written to the developer. You can choose to remove it.